styles.css

```diff
@@ -92,5 +97,214 @@ body {
   margin: 0 auto;
   margin-top: -120px;
-  margin-left: 270px;
+  margin-left: 255px;
+}
+
+.middle-section {
+  width: 375px;
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: 680px 500px 500px 500px 500px 500px;
+  background-color: #fff;
+}
+
+.my-recent {
+  width: 375px;
+}
+
+.recent-img {
+  margin-left: 24px;
+}
+
+.recent-title {
+  display: inline-block;
+  border-bottom: 1px solid black;
+  margin: 0 20px 0 20px;
+  height: 83px;
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+  margin-bottom: 70px;
+}
+
+.recent-title2 {
+  display: inline-block;
+  margin: 0 20px 0 20px;
+  height: 36px;
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+}
+
+.contend {
+  font-size: 16px;
+  padding: 0 24px 0 24px;
+  width: 327px;
+}
+
+.my-recent ul {
+  display: flex;
+  width: auto;
+  justify-content: space-evenly;
+  list-style-type: none;
+  padding: 0;
+}
+
+.lenguajes {
+  font-size: 20px;
+  font-weight: 500;
+  background-color: transparent;
+  padding: 10px 12px;
+  border: 1px solid  #8993a4;
+  transition: 0.3s;
+}
+
+.lenguajes:hover {
+  background-color: #8993a4;
+}
+
+.lenguajes:active {
+  background-color: #3367c0d0;
+}
+
+.see-style {
+  background-color: #ff6b00;
+  text-decoration: none;
+  height: 24px;
+  width: 100px;
+  padding: 7px 24px 6px 10px;
+  color: #fff;
+  font-family: 'Roboto', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  transition: 0.3s;
+}
+
+.see-style:hover {
+  background-color: #ff6a0086;
+}
+
+.see-style:active {
+  background-color: #4d4a49be;
+}
+
+.bak-image {
+  display: flex;
+  flex-direction: column;
+  background-image: url("pictures/Img\ Placeholder\ \(1\).svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.works {
+  color: #fff;
+}
+
+.works-title {
+  width: 295px;
+  font-size: 32px;
+  font-weight: 400;
+  margin: 100px 16px 0 16px;
+  line-height: 44px;
+}
+
+.works-contend {
+  width: 295px;
+  font-size: 16px;
+  margin: 0 16px 0 16px;
+}
+
+.works ul {
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  list-style-type: none;
+  padding: 40px 80px 10px 40px;
+}
+
+.works li {
+  font-size: 20px;
+  font-weight: 500;
+  background-color: rgba(197, 192, 192, 0.3);
+  padding: 10px 12px;
+  border: 1px solid  #8993a4;
+  transition: 0.3s;
+}
+
+.works li:hover {
+  background-color: rgba(150, 143, 143, 0.89);
+}
+
+.works li:active {
+  background-color: rgb(214, 149, 64);
+}
+
+.proyect-button {
+  margin: 30px px 0 0;
+  display: inline-block;
+  text-align: center;
+  background-color: #ff6b00;
+  text-decoration: none;
+  height: 24px;
+  padding: 9px 24px 7px 10px;
+  color: #fff;
+  font-family: 'Roboto', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  transition: 0.3s;
+}
+
+.proyect-button:hover {
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you add the correct button interactions including hover and active (when it's pressed), as shown in the Figma template below: 

<img width="541" alt="image" src="https://user-images.githubusercontent.com/52934128/193900851-3cb52cfc-3deb-4bfa-9ad8-054f39c6ea76.png">

the button on the pressed state has the `background: #E05304`  but yours has a wrong color of `#242323de` and on hover has the `background: #FFF5E1` but your has a wrong color of `#ff6a0060`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #E05304;
}
```
**for hover state**
```
button:hover {
    background-color: #FFF5E1;
}
````